coalib/bears/Bear.py

```diff
@@ -74,6 +74,17 @@ class Bear(Printer, LogPrinter):
     ...     REQUIREMENTS = PythonRequirement.multiple(
     ...         ('colorama', '0.1'), 'coala_decorators')
 
+    To specify additional attributes to your bear, use the following:
+    
+    >>> class SomeBear(Bear):
+    ...     AUTHOR = "Jon Snow"
+    ...     AUTHOR_EMAIL = "jon_snow@gmail.com"
+    ...     MAINTAINER = ("Catelyn Stark",)
+    ...     MAINTAINER_EMAIL = ("catelyn_stark@gmail.com",)
```

Comment:
maintainerS.

Also you want to have a property for this which always makes it a list, just like we do with LANGUAGES